peachjam_api/views.py

```diff
@@ -127,3 +131,44 @@ def post(self, request):
                 )
                 return Response(html)
         return Response()
+
+
+class DocumentProblemView(APIView):
+    permission_classes = []
+    allowed_methods = ["post", "get"]
+
+    def post(self, request):
+        document_link = request.POST.get("document_link")
+        problem_description = request.POST.get("problem_description")
+        email_address = request.POST.get("email_address")
+
+        context = {
+            "document_link": document_link,
+            "problem_description": problem_description,
+        }
+
+        plain_txt_msg = render_to_string(
+            "peachjam/emails/document_problem_email.txt",
+            context=context,
+        )
+
+        message = render_to_string(
+            "peachjam/emails/document_problem_email.html", context=context
+        )
+
+        recipient_list = [settings.DEFAULT_FROM_EMAIL]
+        if email_address:
+            recipient_list.append(email_address)
+
+        return (
+            render(request, "peachjam/emails/document_issue_email_success.html")
+            if send_mail(
```

Comment:
It's uncommon to use the inline `if` syntax when the condition is a function call -- it makes it difficult to read and debug. Rather call `send_mail()` and store the result in a variable.